deployments/cloudfoundry/bosh/DEVELOPMENT.MD

```diff
@@ -0,0 +1,137 @@
+# Development Guide
```

Comment:
From the latest link checker job: https://github.com/signalfx/splunk-otel-collector/runs/6119762127?check_suite_focus=true
```
[EXCLUDED] file:///github/workspace/deployments/cloudfoundry/bosh/DEVELOPMENT.MD | Excluded
```
Let's rename this file to `DEVELOPMENT.md` (lower-case `.md`).

Reply:
Updated, thanks!